src/System.Configuration.ConfigurationManager/System.Configuration.ConfigurationManager.sln

```diff
@@ -30,10 +36,10 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{B7697463-7C98-4462-BA09-67B7BF3842B6}.Debug|Any CPU.ActiveCfg = Windows_Debug|Any CPU
```

Comment:
You will need to sync the changes to this file with @karajas recently merged PR.